knora-ontologies/standoff-onto.ttl

```diff
@@ -97,7 +99,19 @@
                               owl:maxCardinality "1"^^xsd:nonNegativeInteger
                            ] ;
 
-                        rdfs:comment "Represents a hyperlink in a text"@en .
+            rdfs:comment "Represents a hyperlink in a text"@en .
+
+###  http://www.knora.org/ontology/standoff#StandoffBlockquoteTag
+
+:StandoffBlockquoteTag rdf:type owl:Class ;
+
+            rdfs:subClassOf :StandoffHyperlinkTag ,
```

Comment:
or otherwise you can just make it a subclass of hyperlink without redefining the attribute.